ftdetect/minisnip.vim

```diff
@@ -0,0 +1,13 @@
+" Vim ftdetect file
+" Language:      minisnip
+" Created by:    Robert Hicks <gowther@zoho.com>
+" Last Change:   2017-08-20
+"
+" This simply sets all files that are in the defined
+" directory as 'minisnip'.
+"
+
+" get the snippets directory
+let snippet_dir = expand(g:minisnip_dir) . "*"
+
+execute 'autocmd BufRead,BufNewFile ' . snippet_dir . ' set filetype=minisnip'
```

Comment:
I am not sure either. I will investigate. It works as is on macOS. I have to add the g:minisnip_dir on Windows. I am only using vim/gvim to test. I see you are using nvim so hopefully that doesn't deviate here too.